stream_alert/rule_processor/threat_intel.py

```diff
@@ -305,7 +305,7 @@ def _segment(ioc_collections):
         """
         result = []
         end = len(ioc_collections)
-        LOGGER.info('[Threat Inel] Rule Processor queries %d IOCs', end)
+        LOGGER.debug('[Threat Inel] Rule Processor queries %d IOCs', end)
```

Comment:
Can we move this debug statement before the caller to `_query`? I'm not sure it provides a lot of value in this function since

Reply:
It will be same thing. `_process_ioc` is the caller to both `_segment` and `_query`. 
This debug info will be helpful when infrastructure is in a small scale, or does testing in AWS testing account.